lsp-ui-doc.el

```diff
@@ -331,7 +415,9 @@ START-X is the position x of the current window.
 START-Y is the position y of the current window."
   (-let* (((x . y) (--> (bounds-of-thing-at-point 'symbol)
                         (nth 2 (posn-at-point (car it)))))
-          (mode-line-y (lsp-ui-doc--line-height 'mode-line))
+          (mode-line-y (if lsp-ui-doc-use-webkit
+                           0
+                         (lsp-ui-doc--line-height 'mode-line)))
```

Comment:
Why change this with webkit only ? I think it should keep the same behavior than native frame. no ?